src/sortedDict.jl

```diff
@@ -524,9 +524,11 @@ end
 ## ordinary data, 2 if it points to the before-start location and 3 if
 ## it points to the past-end location.
 
-status{K, D, Ord <: Ordering}(i::SDToken{K,D,Ord}) = 
-       !(i.semitoken.address in i.container.bt.useddatacells)? 0 :
-                        (i.semitoken.address == 1? 2 : (i.semitoken.address == 2? 3 : 1))
+function status{K, D, Ord <: Ordering}(i::SDToken{K,D,Ord})
+    i.semitoken.address == 0 && return 0
+    !(i.semitoken.address in i.container.bt.useddatacells)? 0 :
+        (i.semitoken.address == 1? 2 : (i.semitoken.address == 2? 3 : 1))
+end
```

Comment:
Can you add a test for this fix?  

@StephenVavasis, does this look right to you?